gradle.properties

```diff
@@ -1,7 +1,7 @@
 kotlin.code.style=official
 
 #Gradle
-org.gradle.jvmargs=-Xmx2048M -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx2048M"
+org.gradle.jvmargs=-Xmx4096M -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx4096M"
```

Comment:
Document why bumping memory was needed (compiling Wasm bundle for tests experienced too high GC overhead).